ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -116,6 +117,16 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		sortBar?.updateSortMethod()
 
         tableView.tableHeaderView = sortBar
+
+		searchController.searchResultsUpdater = self
+		searchController.obscuresBackgroundDuringPresentation = false
+		searchController.hidesNavigationBarDuringPresentation = true
+		searchController.searchBar.placeholder = "Search".localized
+		navigationItem.searchController = searchController
+		navigationItem.hidesSearchBarWhenScrolling = false
```

Comment:
I noticed Files app has the search bar present all the time, but I'm not sure it's also the right choice for us. It's not a code issue, of course, so this can stay as is. We should just track this in an issue maybe, and discuss it there.

Reply:
I've just copied the Files app design. Yes, we could open an issue and talk about it.
@michaelstingl  what do you think?

Reply:
Fine for first iteration 👍 Let’s wait for user feedback and change later…